cmd/restore.go

```diff
@@ -123,13 +128,23 @@ func (c *restoreCommand) Run(ctx *cmd.Context) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+
+	username := c.username
+	password := c.password
+	if c.username == "" {
+		username, password, err = c.loadCreds()
+		if err != nil {
+			return errors.Annotate(err, "reading credentials from agent.conf")
```

Comment:
if the underyling error is 
"couldn't find an agent.conf - please specify username and password"

this annotation will read a bit icky.

Maybe make all errors from ReadCredsFromAgentConf well worded and avoid the annotation

Reply:
How about I change the annotation to "loading credentials"? Then the annotated error would be "loading credentials: couldn't find an agent.conf - please specify username and password", which reads ok.

Reply:
sgtm